apps/prairielearn/src/sync/course-db.js

```diff
@@ -1034,6 +1035,15 @@ async function validateQuestion(question) {
     }
   }
 
+  if (question.externalGradingOptions?.timeout) {
+    if (question.externalGradingOptions.timeout > config.externalGradingMaximumTimeout) {
+      question.externalGradingOptions.timeout = config.externalGradingMaximumTimeout;
+      warnings.push(
+        `External grading timeouts of more than ${config.externalGradingMaximumTimeout} seconds are not allowed. Timeout has been set to ${config.externalGradingMaximumTimeout} seconds.`,
```

Comment:
Maybe we could print the requested value in the error message? I also tried to soften the "not allowed" phrasing.
```suggestion
        `External grading timeout value of ${question.externalGradingOptions.timeout} seconds exceeds the maximum value and has been limited to ${config.externalGradingMaximumTimeout} seconds.`,
```